ts/features/fci/analytics/index.ts

```diff
@@ -97,19 +97,18 @@ export const trackFciUxSuccess = (
     doc_signed_count,
     sign_count,
     optional_sign_count,
```

Comment:
We should call them `doc_signed_count`, `signed_count` and `optional_signed_count` to follow the spreadsheet, what do you think?

Reply:
Good catch! There is a Jira ticket to fix, so it will be fixed in another PR.